### ISSUE TITLE
Drawio >=21.5.0 crashing on wayland

### ISSUE THREAD

**Issue by aacebedo** [NixOS/nixpkgs#243934]
### Describe the bug

A segfault occurs when trying to start drawio on wayland.
This seems to be due to the use of electron 25 as replacing it with electron 24 fixes the issue.

### Steps To Reproduce
1. Run drawio on a wayland setup

### Expected behavior
The program shall run.

### Additional context
This bug is mentioned in #239585. Other electron based packages are also impacted (#238416)
@DarkOnion0 @alyssais 

### Metadata
```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.3.9-xanmod1, NixOS, 23.11 (Tapir), 23.11.20230709.408c0e8`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.15.1`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by DarkOnion0** [comment]
Thanks for opening this separate issue, but don't forget to mention `@alyssais`, the 2nd maintainers of the drawio package :sweat_smile:.

Personally, I think we should wait on https://github.com/NixOS/nixpkgs/issues/238416#issuecomment-1634245825 answer to decide what change should be made on the package.

For now, you can enable XWayland (I presume it isn't) or just overriding the electron package version as you mentioned as a temporary workaround.

**Comment by aacebedo** [comment]
> Thanks for opening this separate issue, but don't forget to mention `@alyssais`, the 2nd maintainers of the drawio package sweat_smile.
> 
> Personally, I think we should wait on [#238416 (comment)](https://github.com/NixOS/nixpkgs/issues/238416#issuecomment-1634245825) answer to decide what change should be made on the package.
> 
> For now, you can enable XWayland (I presume it isn't) or just overriding the electron package version as you mentioned as a temporary workaround.

Sorry I changed the pings :)
I've overridden electron as I prefer to XWayland as much as possible.

**Comment by DarkOnion0** [comment]
Recently a _global_ issue about this problem was created, you can now also check #244742 for further informations / tips

**Comment by DarkOnion0** [comment]
Hi, it's been quite a while now, but could you please recheck that the problem you were facing is still happening with the latest version (not merged yet, see #278499). On my side, I tried (and successfully) launched the app with `NIXOS_OZONE_WL=1` (since I'm using XWayland)!

**Comment by aacebedo** [comment]
> Hi, it's been quite a while now, but could you please recheck that the problem you were facing is still happening with the latest version (not merged yet, see #278499). On my side, I tried (and successfully) launched the app with `NIXOS_OZONE_WL=1` (since I'm using XWayland)!

Confirmed
